### PR TITLE
Rename Monitoring bundle to Observability plugin.

### PR DESCRIPTION
**Pull request by mattmoor**
Related: https://github.com/knative/docs/issues/2265

/assign @ellenevans @RichieEscarez

### CODE REVIEW

**Comment by RichieEscarez** [comment]
awesome, thanks Matt! 

one question, does this content align with the existing instructions here: https://knative.dev/development/serving/installing-logging-metrics-traces/

do we cross link them?

**Comment by mattmoor** [comment]
I haven't touched that, and if it worked before it should still work.

**Comment by mattmoor** [comment]
The link check is non-blocking and upset by a link with a hugo shortcode embedded :(

File: docs/install/any-kubernetes-cluster.md [hunk 1]
```diff
@@ -587,13 +587,13 @@ To learn more about the Cloud Audit Logs source, try [our sample](../eventing/sa
 You can find a number of samples for Knative Eventing [here](../eventing/samples/README.md). A quick-start guide is available [here](../eventing/getting-started.md).
 
 
-## Installing the Monitoring bundle
+## Installing the Observability plugin
 
 {{< feature-state version="v0.2" state="alpha" >}}
 
-Knative provides a bundle of monitoring components that can be used to make the Serving and Eventing components more observable.
+Knative provides a collection of observability components that can be used to make the Serving and Eventing components more observable.
```
Comment (RichieEscarez) [inline docs/install/any-kubernetes-cluster.md]:
```suggestion
Install the following observability features to enable logging, metrics, and request tracing in your Serving and Eventing components.
```

File: docs/install/any-kubernetes-cluster.md [hunk 2]
```diff
@@ -587,13 +587,13 @@ To learn more about the Cloud Audit Logs source, try [our sample](../eventing/sa
...
 
-Before you install any specific monitoring components, you must first install the core monitoring pieces:
+Before you install any specific observability components, you must first install the core:
```
Comment (RichieEscarez) [inline docs/install/any-kubernetes-cluster.md]:
```suggestion
 All observibility plugins require that you first install the core:
```

File: docs/install/any-kubernetes-cluster.md [hunk 3]
```diff
@@ -587,13 +587,13 @@ To learn more about the Cloud Audit Logs source, try [our sample](../eventing/sa
...
 
 ```bash
 kubectl apply --filename {{< artifact repo="serving" file="monitoring-core.yaml" >}}
```
Comment (RichieEscarez) [inline docs/install/any-kubernetes-cluster.md]: Can we add below this:

"After the core is installed, you can choose to install one or all of the following observability plugins:"

**Pull request merged**